src/main/resources/fabric.mod.json

```diff
@@ -28,7 +29,7 @@
   "depends": {
 	"fabricloader": ">=0.7.4",
 	"fabric": "*",
-	"minecraft": "1.17.x"
+	"minecraft": "1.18.2"
```

Comment:
i prefer to use .x because that makes it usable for all versions, not just the latest